src/components/app/details/appConfig/AppComposeRouter.tsx

```diff
@@ -124,6 +127,20 @@ export default function AppComposeRouter({
         )
     }
 
+    const defaultRoute = (): string => {
+        if (isUnlocked.gitOpsConfig) {
+            return URLS.APP_GITOPS_CONFIG
+        } else if (isUnlocked.deploymentTemplate) {
+            return URLS.APP_DEPLOYMENT_CONFIG
+        } else if (isUnlocked.dockerBuildConfig) {
+            return URLS.APP_DOCKER_CONFIG
+        } else if (isUnlocked.workflowEditor) {
+            return URLS.APP_WORKFLOW_CONFIG
+        } else {
+            return URLS.APP_GITOPS_CONFIG
```

Comment:
It should be `APP_GIT_CONFIG`. Please verify and fix

Reply:
Verified & Resolved